app/src/main/java/org/fossasia/phimpme/gallery/activities/SplashScreen.java

```diff
@@ -40,13 +43,13 @@
     public final static int ALBUMS_PREFETCHED = 23;
     public final static int PHOTOS_PREFETCHED = 2;
     public final static int ALBUMS_BACKUP = 60;
-    private boolean PICK_INTENT = false;
+    private static boolean PICK_INTENT = false;
     public final static String ACTION_OPEN_ALBUM = "vn.mbm.phimp.leafpic.OPEN_ALBUM";
 
     //private HandlingAlbums albums;
-    private Album album;
-    private boolean can_be_finished = false;
-    private Intent nextIntent = null;
+    private static Album album = null;
```

Comment:
Why do you need to make this static? You can use the weak reference in the AsyncTask, right?
@codedsun @ashley8jain @batbrain7 @raj10071997

Reply:
Should I pass the variables as parameters and access using WeakReference Class?

Reply:
@mayank8318 @DhruvamSharma check gitter, yes we can :( my fault! I approved